scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/JobInfo.java

```diff
@@ -305,4 +305,16 @@
      * @return non-empty result map test
      */
     boolean isResultMapPresent();
+
+    /**
+     * Return the parent id of the job
+     * @return parent id of the job, or null if the job as no parent
```

Comment:
@return parent id of the job, or null if the job **has** no parent

Reply:
ok